flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -102,39 +92,14 @@ public void setUp() throws Exception {
 		config.setString(JobManagerOptions.ADDRESS, "localhost");
 		config.setInteger(JobManagerOptions.PORT, freePort);
 		config.setString(AkkaOptions.ASK_TIMEOUT, AkkaOptions.ASK_TIMEOUT.defaultValue());
-
-		try {
-			scala.Tuple2<String, Object> address = new scala.Tuple2<String, Object>("localhost", freePort);
-			jobManagerSystem = AkkaUtils.createActorSystem(config, new scala.Some<scala.Tuple2<String, Object>>(address));
-		}
-		catch (Exception e) {
-			e.printStackTrace();
-			fail("Setup of test actor system failed.");
-		}
-	}
-
-	@After
-	public void shutDownActorSystem() {
-		if (jobManagerSystem != null) {
-			try {
-				jobManagerSystem.terminate();
-				Await.ready(jobManagerSystem.whenTerminated(), Duration.Inf());
-			} catch (Exception e) {
-				e.printStackTrace();
-				fail(e.getMessage());
-			}
-		}
 	}
 
 	/**
 	 * Tests that invalid detached mode programs fail.
 	 */
 	@Test
 	public void testDetachedMode() throws Exception{
-		jobManagerSystem.actorOf(
-			Props.create(SuccessReturningActor.class),
-			JobMaster.JOB_MANAGER_NAME);
-		StandaloneClusterClient out = new StandaloneClusterClient(config);
+		final ClusterClient<?> out = new MiniClusterClient(new Configuration(), MINI_CLUSTER_RESOURCE.getMiniCluster());
```

Comment:
It was called `out` before but I don't understand why. I think `clusterClient` is a better name.